crates/sound/src/audio_backends/pw_backend.rs

```diff
@@ -400,19 +403,47 @@ impl AudioBackend for PwBackend {
         if let Some(stream) = stream_hash.get(&stream_id) {
```

Comment:
I am somewhat not sure about this,  as calling the release function with a stream_id that already got disconnected will crash with `could not disconnect stream` :) instead you also have to make sure to remove the stream_id from stream/hash & stream_listener

Reply:
If you remove the item from the map, the std::mem::drop call would no longer be needed as it is automatically done